Library/Homebrew/formula.rb

```diff
@@ -1063,6 +1063,18 @@ def std_cmake_args
     ]
   end
 
+  # Returns the path to a make 3.81 or newer binary.
+  # Returns the system make on Leopard and newer, and the
+  # path to the actually-installed make on Tiger or older.
+  def make_path
```

Comment:
Leopard uses superenv, but Tiger doesn't yet. I did a few experiments, but was limited by a) system Ruby version, b) Tiger not shipping `xcrun`. I'm planning to try to reenable it once I have Tigerbrew using Ruby 2.0.